release: v0.2.2 (#70)
ansys/pysimai: CHANGELOG.rst

Changelog
---------

0.2.2 (2024-07-17)
******************

New:

> - Added support for the custom volume point cloud postprocessing. Use :py:meth:`Geometry.upload_point_cloud<ansys.simai.core.data.geometries.Geometry.upload_point_cloud>` to upload a point cloud file on a geometry and run the post processing through :py:meth:`Prediction.post.custom_volume_point_cloud<ansys.simai.core.data.post_processings.PredictionPostProcessings.custom_volume_point_cloud` to run the postprocessing.

Reviewer: Post-processing custom volume of point cloud is now supported. 
```suggestion
- Post-processing custom volume of point cloud is now supported. Clients can use :py:meth:`Geometry.upload_point_cloud<ansys.simai.core.data.geometries.Geometry.upload_point_cloud>` to upload a point cloud file on a geometry and :py:meth:`Prediction.post.custom_volume_point_cloud<ansys.simai.core.data.post_processings.PredictionPostProcessings.custom_volume_point_cloud` to run the postprocessing.
```
